.github/workflows/zebra-test-ci.yaml

```diff
@@ -27,7 +27,8 @@ jobs:
         uses: actions/checkout@v3
         with:
           repository: QED-it/zebra
-          ref: ${{ vars.ZEBRA_BRANCH_TO_TEST || 'zsa-integration-demo' }} # The branch in Zebra we want to test against
+          # FIXME: Change to: ref: ${{ vars.ZEBRA_BRANCH_TO_TEST || 'zsa-integration-demo-with-state' }} after this PR is verified.
+          ref: zsa-integration-demo-with-state
```

Comment:
revert to the original config once https://github.com/QED-it/zebra/pull/89#pullrequestreview-3410377112 is merged